### ISSUE TITLE
uSkyBlock is not detecting .schem files

### ISSUE THREAD

**Issue by lpjn** [rlf/uSkyBlock#1207]
_Please paste the output from `/usb version` below_
```
[20:00:39 INFO]: lpjn issued server command: /usb version
>usb version
[20:01:40 INFO]: Name: uSkyBlock
[20:01:40 INFO]: Version: 2.7.8-SNAPSHOT
[20:01:40 INFO]: Description: Ultimate SkyBlock v2.7.8-SNAPSHOT-1b90f8d
[20:01:40 INFO]: Language: en (en)
[20:01:40 INFO]:   State: d=128, r=64, i=1, p=2, n=true, awe=true
[20:01:40 INFO]: Server: CraftBukkit git-Spigot-1a3504a-84f3da3 (MC: 1.13.2)
[20:01:40 INFO]:   State: online=true, bungee=false
[20:01:40 INFO]: ------------------------------
[20:01:40 INFO]: Vault 1.7.2-b107 (ENABLED)
[20:01:40 INFO]: WorldEdit 1.13.163 (WRONG-VERSION)
[20:01:40 INFO]: WorldGuard 7.0.0-beta-03;e51a220 (ENABLED)
[20:01:40 INFO]: FastAsyncWorldEdit 1.13-breaking (ENABLED)
[20:01:40 INFO]: Multiverse-Core 4.0.0-b766 (ENABLED)
[20:01:40 INFO]: Multiverse-Portals 4.0.0-b799 (ENABLED)
[20:01:40 INFO]: ------------------------------
```

_What steps will reproduce the problem?_
1. Create schematic using 1.13 worldedit (specifically using FAWE but that does not make a difference)
2. Place schematic in schematics folder and correctly reference it in config
3. Plugin ignores it on GUI and creation
4. Renaming file to .schematic causes plugin to pick it up but then file corrupts
5. It appears the plugin does not like .schem files, which are the WorldEdit default.

_If you have any log-files, please paste them to [pastebin.com](http://pastebin.com)_
* server-log: There are no errors.

(Not following the template will result in closing the issue.)

**Comment by lpjn** [comment]
If you would like my config / island files so you can test, I have attached them

**Comment by BlackBeltPanda** [comment]
For the time being, you can rename the file extension from ".schem" to ".schematic" and they should work.

**Comment by BlackBeltPanda** [comment]
Renaming from ".schem" to ".schematic" should work. I believe there's a bug with FAWE at the moment that pastes it in looking corrupt when set to paste from memory. The fix for that is to open FAWE's config and change "clipboard.use-disk" to "true".

**Comment by lpjn** [comment]
Hi,

These settings are already set:

	Line 25:   # Store the clipboard on disk instead of memory
	Line 28:   use-disk: true
	Line 30:   #  - TODO: Buffered random access with compression is not implemented on disk yet
	Line 35:   # Number of days to keep history on disk before deleting it
	Line 158:   # Should history be saved on disk:
	Line 163:   use-disk: true

I tried forcing the schematic save to .schematic but FAWE did not support it. Pasting the file after it is changed to a .schematic pastes nothing, blank.

**Comment by BlackBeltPanda** [comment]
That's odd, I had the same issue but switching from memory to disk fixed it for me. Have you tried using memory instead of disk? Just set "use-disk" to false.

**Comment by lpjn** [comment]
Same result I'm afraid! :( Blank clipboard when the file extension is modified - this was a new schematic post cfg change + restart too

**Comment by BlackBeltPanda** [comment]
When you copy the build, before saving it as a schematic, did you get any errors? I had to use "//copy -e" in order to copy without errors. Something about the newer versions of FAWE not copying entities correctly. Maybe try "//copy -e" and then save it as a schematic and rename to ".schematic"?

**Comment by Muspah** [comment]
I'm not sure about the exact cause so I'll have to look into it, but I do know that there is some refactoring to do around the code pasting in schematics. I will move it upwards on my todo list to look into it ;)

**Comment by lpjn** [comment]
No luck with that @BlackBeltPanda - Thanks for the suggestion though!!!

Thanks @Muspah

**Comment by BlackBeltPanda** [comment]
@lpjn My best guess for why renaming isn't working anymore is FAWE may have updated and changed something about the way it saves/loads schematics. For reference, I'm using the 1.13.115 WE build of FAWE which is a bit behind yours. Only thing I can think of, lol.

@Muspah Good luck! The code for pasting schematics has become a bit messy over time, lol.

**Comment by Muspah** [comment]
Yeah, it's not the only thing that became messy, hahaha. Can't blame anyone for it though, well, lets call it a challenge instead of a problem, I've had some management meetings today 😌

**Comment by lpjn** [comment]
I resolved this by using 1.13.115 of FAWE and renaming the file as discussed. Thank you @BlackBeltPanda

**Comment by Muspah** [comment]
The issue with USB not accepting schematics ending in .schem should be fixed with PR #1208 included in the upcoming 2.8.2 release of USB. 2.8.0+ releases are for MC1.14, and given the fact that FAWE has no 1.14 release available yet, I'm going to postpone fixing or working around any FAWE issues until FAWE has a 1.14 release available.
